src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const populations = [...document.querySelectorAll('span.population')];
+// const onePopulation = document.queryElementById
+
+const mappedPopulation = [];
+
+populations.forEach((amount) => {
+  const number = amount.textContent
+    .split('')
+    .filter((el) => el !== ',')
+    .join('');
+
+  mappedPopulation.push(+number);
+});
+
+const formattedNumberUS = (input) => {
+  return input.toLocaleString('en-US');
+};
+
+const total = mappedPopulation.reduce(
+  (accumulator, curentValue) => accumulator + curentValue,
```

Comment:
There is a typo in the parameter name `curentValue`. It should be `currentValue` to correctly represent the current value in the `reduce` method.